hawkular-alerts-actions/hawkular-alerts-actions-plugins/hawkular-alerts-actions-generic/hawkular-alerts-actions-generic-plugin/pom.xml

```diff
@@ -22,13 +22,13 @@
 
   <parent>
     <groupId>org.hawkular.alerts</groupId>
-    <artifactId>hawkular-alerts-actions-plugins</artifactId>
+    <artifactId>hawkular-alerts-actions-generic</artifactId>
```

Comment:
I guess you have caught a good one.
No, webhook is not generic.
I simply moved the snmp plugin to a generic one to show point users to that as a "template" for building a new plugin.
It has not snmp logic it's just a pure receive-and-print example.
But this comment is good as I see that perhaps the artifact name has a missing -plugin suffix.
I will review it.